word_guess.rb

```diff
@@ -0,0 +1,193 @@
+        if @word == "PET"
+          word_pet
+        elsif @word == "HUT"
+          word_hut
+        elsif @word == "CAT"
+          word_cat
+        elsif @word == "DOG"
+          word_dog
+        end
+
+    else
+      puts "#{@user_letter} is not included in the word!"
+      @rose_array.delete_at(0)
+      puts "\n
+      #{@rose_array.join}
+        \\ \\ | / /
+          _\\|/_
+         |_____|
+          |   |
+          |___| "
+      if @rose_array.length == 0
+        loser_method
+      else
+        puts "\nLetters Guessed:\n"
+        puts @letters_guessed.join    # need to do something with @letters_guessed first yo... #of letters user has already guessed.
+        puts "\nPlease choose a letter and press enter!"
+        @user_letter = gets.chomp.upcase
+        puts "You selected #{@user_letter}."
+        include_in_guessed
+      end
+    end
+  end
+
+ def word_pet
+  if @user_letter == "E"
+    @dash_array.fill("E", 1..1)
+  elsif @user_letter == "P"
+    @dash_array.fill("P", 0..0)
+  elsif @user_letter == "T"
+    @dash_array.fill("T", 2..2)
+  end
+  puts "\nLetters Guessed:\n"
+  puts @letters_guessed.join    # need to do something with @letters_guessed first yo... #of letters user has already guessed.
+  puts @dash_array.join
+  if @dash_array.include?(" _ ")
+    puts "\nPlease choose a letter and press enter!"
+    @user_letter = gets.chomp.upcase
+    puts "You selected #{@user_letter}."
+    include_in_guessed
+  else
+   winner_method
+  end
+end
+
+def word_hut
+  if @user_letter == "U"
+    @dash_array.fill("U", 1..1)
+  elsif @user_letter == "H"
+    @dash_array.fill("H", 0..0)
+  elsif @user_letter == "T"
+    @dash_array.fill("T", 2..2)
+  end
+  puts "\nLetters Guessed:\n"
+  puts @letters_guessed.join    # need to do something with @letters_guessed first yo... #of letters user has already guessed.
+  puts @dash_array.join
+  if @dash_array.include?(" _ ")
+    puts "\nPlease choose a letter and press enter!"
+    @user_letter = gets.chomp.upcase
+    puts "You selected #{@user_letter}."
+    include_in_guessed
+  else
+   winner_method
+  end
+end
+
+def word_dog
+  if @user_letter == "O"
+   @dash_array.fill("O", 1..1)
+ elsif @user_letter == "D"
+   @dash_array.fill("D", 0..0)
+  elsif @user_letter == "G"
+    @dash_array.fill("G", 2..2)
+  end
+  puts "\nLetters Guessed:\n"
+  puts @letters_guessed.join    # need to do something with @letters_guessed first yo... #of letters user has already guessed.
+  puts @dash_array.join
+  if @dash_array.include?(" _ ")
+    puts "\nPlease choose a letter and press enter!"
+    @user_letter = gets.chomp.upcase
+    puts "You selected #{@user_letter}."
+    include_in_guessed
+  else
+   winner_method
+  end
+end
+
+def word_cat
+  if @user_letter == "A"
+   @dash_array.fill("A", 1..1)
+ elsif @user_letter == "C"
+   @dash_array.fill("C", 0..0)
+  elsif @user_letter == "T"
+    @dash_array.fill("T", 2..2)
+end
+  puts "\nLetters Guessed:\n"
```

Comment:
Any time you see code duplicated a few times like this - this would be a great place to consider putting the code into a method! The first part of each of these word methods are different, but this bottom part seems to be the same.